Modules/Scripted/Endoscopy/Endoscopy.py

```diff
@@ -704,6 +726,9 @@ def __init__(self, inputCurve, dl=0.5):
         self.dl = dl  # desired world space step size (in mm)
         self.setControlPoints(inputCurve)
 
+    def __del__(self):
+        self.cleanup()
+
```

Comment:
```suggestion
```

The cleanup function is always explicitly called.

Reply:
Thank you, I'll remove that.